src/app/recover.rs

```diff
@@ -0,0 +1,20 @@
+use futures::Stream;
+use linkerd2_error::Error;
+pub use linkerd2_error::Recover;
+pub use linkerd2_exp_backoff::{ExponentialBackoff, ExponentialBackoffStream};
+
+pub fn always(backoff: ExponentialBackoff) -> Always {
```

Comment:
Nit/TIOLI: why is this in `app` rather than in the backoff crate?

Reply:
backoff doesn't have a dependency on on error

Reply:
also, there's no reason this can't use other backoffs in the future, etc.... all of the concrete integrations will generally end up in app if they aren't generic.